projects/plugins/jetpack/extensions/blocks/simple-payments/test/validate.js

```diff
@@ -4,6 +4,16 @@
 import { settings } from '../';
 import runBlockFixtureTests from '../../../shared/test/block-fixtures';
 
+beforeEach(() => {
+    Intl.NumberFormat = jest
+        .fn()
+        .mockImplementation(() => ({format: value => `A$${value.toString()}.00`}));
```

Comment:
Curious that the linter doesn't complain about expected spaces between `() {}` etc

Reply:
that is strange, have fixed now.